{{ cookiecutter.project_slug }}/pyproject.toml

```
@@ -4,8 +4,10 @@ name = "{{ cookiecutter.project_slug }}"
 version = "0.1.0"
 description = "{{ cookiecutter.project_description }}"
 authors = [
-    "{{ cookiecutter.author }} <{{ cookiecutter.author_email }}>",
+    "{{ cookiecutter.author }} <{{ cookiecutter.author_email }}>"
```

I suspect we're going to need a comma here
```suggestion
    "{{ cookiecutter.author }} <{{ cookiecutter.author_email }}>",
```

---

We don't actually need a comma, because one's added on the line below.

---

Oh I see, I did miss that. Won't that format really weirdly? It seems like it will be a single, unindented line with one comma in it? Like:

```toml
uthors = [
    "Author <mail@mail.com>"
,
    "Imperial College London RSE Team <ict-rse-team@imperial.ac.uk>"
]
```

---

jinja is weird...

---

> We don't have to do it this way though if you think it's more readable to have the `if` on the previous line.

Sorry, I missed this reply. No, it's not more or less readable, I just saw the if had been removed previously.